jmix-bom/bom.gradle

```diff
@@ -279,14 +280,77 @@ dependencies {
 
         api 'com.amazonaws:aws-java-sdk-core:1.11.1034'
 
-        def yargVersion = '2.2.21'
-        api "com.haulmont.yarg:yarg-api:$yargVersion"
-        api "com.haulmont.yarg:yarg:$yargVersion"
+        api 'org.apache.httpcomponents:httpclient:4.5.14'
+
+        api 'commons-logging:commons-logging:1.2'
+        api 'org.apache.commons:commons-lang3:3.9'
+        api 'commons-io:commons-io:2.6'
```

Comment:
commons-io is also already exists in this bom file. And 2.6 is not the newest version. We already have 2.11